### PR TITLE
Rfxcom itemstates

### PR DESCRIPTION
**Pull request by juri8**
Make the RFXCom plugin update the itemstates manually if a command was a success. Also fix some threading code.

### CODE REVIEW

File: bundles/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/RFXComBindingProvider.java [hunk 1]
```diff
@@ -21,7 +21,7 @@
  * @author Pauli Anttila
```
Comment (teichsta) [inline bundles/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/RFXComBindingProvider.java]: please add yourself as author

**Comment by teichsta** [comment]
Thanks @juri8, the contribution looks good to me now. Could you finally please squash your commits into one? Best, Thomas E.-E.

**Comment by teichsta** [comment]
if you could finally squash the commits into one i'll be able to merge into master. Thanks, Thomas E.-E.

**Comment by juri8** [comment]
closing in favour of pull request #2414 which contains all this changes squashed

**Pull request closed without merging**